### ISSUE TITLE
Does not run on all XScreens

### ISSUE THREAD

**Issue by ghost** [elliptic-shiho/etwm#4]
I found a post a while back about etwm being a slightly more modernized ctwm (hoping for 32bit depth) so I figured I'd give it a try. Seems the xrandr bits break functionality and limits it to only XScreen:0.0. CTWM runs fine on multiGPU setups but ETWM works but isn't usable.

**Comment by elliptic-shiho** [comment]
I think, are you talking about https://github.com/bbidulock/etwm? I don't know about ctwm. 

This project is just for my practical use, and this project is targeted "minimal modification from original twm's behavior". More, I have never introduced this for someone expressly. 

btw, I didn't realized for 32-bit depth bug. I use this WM now but 24-bit only. thanks for report, I may fix this bug in the future.

**Comment by ghost** [comment]
You are right, I checked and it's the bbidulock in the AUR. Yeah the 24bit depth is kinda odd for what seems to be all the TWM derived variants/forks. Odd in that CTWM had some recent updates but it's still only doing 24bit so I had hoped to find similar functionality that did 32 from another TWM variant.

- Sorry for the bother.
